website/content/docs/auth/login-mfa.mdx

```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+page_title: Multi-Factor Authentication (MFA) for Login - Auth Methods
+description: |-
+  Multi-factor authentication (MFA) is supported for several authentication
+  methods.
+---
+# Login MFA
+
+Vault has support for Multi-factor Authentication (MFA) for authenticating to
+an auth method, using different authentication types. We use the term `Login MFA` to distinguish
```

Comment:
```suggestion
an auth method using different authentication types. We use the term `Login MFA` to distinguish
```

Reply:
Recommendation: when referencing terms, use italics instead of wrapping the term in code.